pkgs/development/haskell-modules/configuration-ghcjs.nix

```diff
@@ -26,7 +28,6 @@ self: super:
 
   # GHCJS does not ship with the same core packages as GHC.
   # https://github.com/ghcjs/ghcjs/issues/676
-  stm = doJailbreak self.stm_2_5_1_0;
```

Comment:
This instead needs to be changed to `stm_2_5_3_0`. ghcjs doesn't ship `stm` as a core package, so this would break other packages that depend on it.

Reply:
Changed to `stm_2_5_3_0`. Thank you!